packages/oqamldebug/oqamldebug.0.9.5/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "sebastien.fricker@gmail.com"
+build: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  ["%{make}%"]
+  ["%{make}%" "install" ]
+]
+remove: [
+  [ "qmake" "CONFIG-=app_bundle" "CONFIG+=release" "PREFIX_BIN=%{bin}%/" "CONFIG-=debug" "oqamldebug.pro" ]
+  [ "make" "uninstall" ]
```

Comment:
I think this needs `make` rather than `"make"` in order to work with systems where `"make"` is not GNU make (`make` is always GNU make).